pages/api/candidateApiClient.module.ts

```diff
@@ -1,36 +1,24 @@
 ﻿import fetch from "node-fetch";
 
-export interface CandidateTestStatus {
-    testId: string;
-    testResult: string;
+export interface SessionCandidate {
+    firstName: string;
+    lastName: string;
+    testStatuses: CandidateTestStatus[];
 }
 
-const baseUrl = `https://localhost:5001`;
-
-export async function getCandidateTestResults() {
-    try {
-        const result = await fetch(
-            //TODO: placeholder endpoint
-            `https://testswitch-api-staging.herokuapp.com/candidates`
-        );
-        const data = await result.json();
-        //TODO: configure for future api call to candidate results, this is currently set to the candidates endpoint
-        return data.items;
-    } catch (error) {
-        console.error(error);
-        return error.message;
-    }
+export interface CandidateTestStatus {
+    testName: string;
+    testStatus: string;
 }
 
-export async function getCandidateTests() {
+export async function getSessionCandidate(token: string): Promise<SessionCandidate> {
+    const baseUrl = `https://testswitch-api-staging.herokuapp.com/sessions/`;
     try {
         const result = await fetch(
-            //TODO: placeholder endpoint
-            `https://testswitch-api-staging.herokuapp.com/candidates`
+            `${baseUrl}${token}`
```

Comment:
I _think_ the base URL probably shouldn't include `sessions`.
Could we change this to

`${baseUrl}/sessions/${token}`?

Reply:
done. ✔️